doc/api/net.md

```diff
@@ -1028,11 +1029,6 @@ then returns the `net.Socket` that starts the connection.
 <!-- YAML
 added: v0.5.0
 -->
-* `options` {Object}
-* `connectionListener` {Function}
-
-Creates a new TCP or [IPC][] server.
-
```

Comment:
Nit: it seems we usually have an empty line after a YAML block.

Reply:
fixed